src/sentry/workflow_engine/models/incident_groupopenperiod.py

```diff
@@ -0,0 +1,32 @@
+from django.db.models import UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    DefaultFieldsModel,
+    FlexibleForeignKey,
+    region_silo_model,
+)
+
+
+@region_silo_model
+class IncidentGroupOpenPeriod(DefaultFieldsModel):
+    """
+    A lookup model for incidents and group open periods.
+    """
+
+    __relocation_scope__ = RelocationScope.Excluded
+
+    incident_id = BoundedBigIntegerField(null=True)
+    group_open_period = FlexibleForeignKey("sentry.GroupOpenPeriod")
```

Comment:
do each of these need `unique=True`? so incident_ids and group_open_periods don't appear multiple times in their respective columns?

Reply:
That's what the unique constraint is for unless I am misunderstanding

Reply:
sounds like setting multiple [fields](https://docs.djangoproject.com/en/5.2/ref/models/constraints/#fields) in the UniqueConstraint is the equivalent of 'unique together', which is slightly different from saying each individual incident can only be present in the column once

Reply:
Ah ok so I will add `unique=True` to these then, ty